src/conchat.js

```diff
@@ -128,6 +145,54 @@ class Con {
     })();
   }
 
+  createDebugRoom(roomName) {
+    if (this.#isStarted()) {
+      console.log('🚫 con.chat()을 실행해주세요.');
+
+      return;
+    }
+
+    if (!this.#hasUsername) {
```

Comment:
앞에 한번 부정 연산자를 두는 것보다는 
```suggestion
    if (!this.#hasUsername) {
    if(this.#hasNotUsername) {
```
이렇게 사용하는게 이중 부정을 방지해서 가독성을 향상시킬 수 있습니다.

Reply:
피드백 감사합니다!
제가 hasUsername 변수를 사용한 이유는 이미 코드 전반에서 hasUsername이 사용되고 있어 
새로운 변수 hasNotUsername를 만들게 된다면 일관성을 해친다고 생각했습니다.
그래서 기존 변수를 활용해서 일관성을 유지하는 게 더 좋다고 생각하여 기존 변수명으로 적용했는데요!
기존 변수가 없다면 말씀해주신 이중 부정을 방지해서 가독성을 향상한다는 점을 생각하면서 변수를 만들면 좋을 것 같습니다!

Reply:
제 의견은 오히려 처음부터 부정형 변수를 사용한다면 그 부정형 변수의 반대값이 필요하다면..? 
그때는` !hasNotUsername` 이런식으로 이중 부정이 되어버려서 클린 코드에서 지양하는 방식이 될 것 같습니다. 
긍정형, 부정형 둘 다 쓰일 수 있는 여지가 있는 내용이라면 긍정형을 우선적으로 사용하는 것이 좋다고 생각합니다.